web/beacon-app/src/utils/inputSanitzer.ts

```diff
@@ -0,0 +1,14 @@
+import DOMPurify from 'dompurify';
+
+export const inputSanitizer = (input: string) => {
+  //  prevent XSS attacks
+  const sanitizedInput = DOMPurify.sanitize(input);
+  // prevent SQL injection
+  const sanitizedSqlInjection = sanitizedInput.replace(/'/g, "\\'");
```

Comment:
Before this is used, I think that we should add more tests with different SQL queries, to maximize the test coverage. This can be handled separately, in the future.